### ISSUE TITLE
Java 9 support.

### ISSUE THREAD

**Issue by mibby** [NuVotifier/NuVotifier#33]
NuVotifier v2.3.5

```
[20:47:19] [Server thread/INFO]: [Votifier] Enabling Votifier v2.3.5
[20:47:19] [Server thread/ERROR]: Error occurred while enabling Votifier v2.3.5 (Is it up to date?)
java.lang.NoClassDefFoundError: javax/xml/bind/DatatypeConverter
	at com.vexsoftware.votifier.net.protocol.v1crypto.RSAIO.load(RSAIO.java:75) ~[?:?]
	at com.vexsoftware.votifier.NuVotifierBukkit.onEnable(NuVotifierBukkit.java:182) ~[?:?]
	at org.bukkit.plugin.java.JavaPlugin.setEnabled(JavaPlugin.java:264) ~[patched_1.12.2.jar:git-Paper-1236]
	at org.bukkit.plugin.java.JavaPluginLoader.enablePlugin(JavaPluginLoader.java:316) [patched_1.12.2.jar:git-Paper-1236]
	at org.bukkit.plugin.SimplePluginManager.enablePlugin(SimplePluginManager.java:404) [patched_1.12.2.jar:git-Paper-1236]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugin(CraftServer.java:386) [patched_1.12.2.jar:git-Paper-1236]
	at org.bukkit.craftbukkit.v1_12_R1.CraftServer.enablePlugins(CraftServer.java:335) [patched_1.12.2.jar:git-Paper-1236]
	at net.minecraft.server.v1_12_R1.MinecraftServer.t(MinecraftServer.java:442) [patched_1.12.2.jar:git-Paper-1236]
	at net.minecraft.server.v1_12_R1.MinecraftServer.l(MinecraftServer.java:403) [patched_1.12.2.jar:git-Paper-1236]
	at net.minecraft.server.v1_12_R1.MinecraftServer.a(MinecraftServer.java:341) [patched_1.12.2.jar:git-Paper-1236]
	at net.minecraft.server.v1_12_R1.DedicatedServer.init(DedicatedServer.java:292) [patched_1.12.2.jar:git-Paper-1236]
	at net.minecraft.server.v1_12_R1.MinecraftServer.run(MinecraftServer.java:616) [patched_1.12.2.jar:git-Paper-1236]
	at java.lang.Thread.run(Thread.java:844) [?:?]
Caused by: java.lang.ClassNotFoundException: javax.xml.bind.DatatypeConverter
	at java.net.URLClassLoader.findClass(URLClassLoader.java:466) ~[?:?]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:105) ~[patched_1.12.2.jar:git-Paper-1236]
	at org.bukkit.plugin.java.PluginClassLoader.findClass(PluginClassLoader.java:90) ~[patched_1.12.2.jar:git-Paper-1236]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:563) ~[?:?]
	at java.lang.ClassLoader.loadClass(ClassLoader.java:496) ~[?:?]
	... 13 more
[20:47:19] [Server thread/INFO]: [Votifier] Disabling Votifier v2.3.5
[20:47:19] [Server thread/INFO]: [Votifier] Votifier disabled.
```

**Comment by Ichbinjoe** [comment]
I will fix this tomorrow morning, thanks for bringing it to my attention.

**Comment by lifehome** [comment]
Any updates on this? Still seeing crashes...

**Comment by mibby** [comment]
Bump. ^^

**Comment by Ichbinjoe** [comment]
I do not have the time to work on it. The time I spend to develop this plugin is for free, and I have too many other obligations which I do for work which must be completed before I even think of addressing this or any other issue on this plugin. If someone has a PR, I will accept it.

**Comment by lifehome** [comment]
Dirty patched it a bit for bukkit and bungee on 1.12, have tested bungee on my production server but not bukkit. Please try out and report bugs if you are interested in the Java 9 build.

https://github.com/lifehome/NuVotifier/releases/tag/v2.3.5
